src/content/docs/accounts/accounts/automated-user-management/okta-scimsso-application-configuration.mdx

```diff
@@ -62,7 +62,8 @@ Assigning users is done using two different tabs in the app. We recommend having
 2. From the **Assignments** form, click on **Assign**.
 3. From the pop up menu, click on **Assign to groups**.
 4. From the **Assign ... to groups** form, click on **Assign** for the group you wish to assign to the application.
-5. Optional: in the **Time zone** field, enter the default time zone for members of the group. Members without a time zone configured, will use the group time zone. Time zone affects how date/times are shown in New Relic. Time zone is specified in IANA Time Zone database format, also known as the "Olson" time zone database format (e.g., "America/Los_Angeles").
+5. **Highly recommended**: We highly recommend setting your users' time zone in Okta. Time zone affects how date/times for that user are shown in New Relic. Users without a time zone configured will be shown in UTC time in New Relic. Time zone is specified in IANA Time Zone database format, also known as the "Olson" time zone database format (e.g., "America/Los_Angeles"). There are several ways in Okta to assign users' time zone, so consult the Okta docs for more information if needed. Here is one way to do this in the **Assignments** tab: 
```

Comment:
I wonder if you might want to remove the language `We highly recommend setting ...` and change to `Set...` since the boldface text already states it's highly recommended?

Reply:
Thank you!